core/services/synchronization/stats_pusher.go

```diff
@@ -224,13 +243,11 @@ func createSyncEventWithStatsPusher(sp *StatsPusher, orm *orm.ORM) func(*gorm.Sc
 		event := models.SyncEvent{
 			Body: string(bodyBytes),
 		}
-		err = scope.DB().Save(&event).Error
+		err = scope.DB().Create(&event).Error
```

Comment:
What's the difference between `Create` and `Save` here?

Reply:
Create only creates new records, Save can update a record if it already exists - which it never should.